packages/logger-plugin/src/logger.plugin.ts

```diff
@@ -87,4 +88,10 @@ export class NgxsLoggerPlugin implements NgxsPlugin {
 
     return ms_ie;
   }
+
+  private _hasPayload(event: any) {
+    const nonEmptyProperties = Object.entries(event).filter(entry => !!entry[1]);
```

Comment:
`entry[1]` ? 

can give a meaningful name

Reply:
`property` maybe?

Reply:
I just always get embarrassed when people go to the index, why the first index?

Reply:
entry returns `[ propertyName, 'propertyValue' ]`

Reply:
prefer

```ts
const nonEmptyProperties = Object.entries(event).filter(([ value ]) => !!value);
```

Reply:
@splincode `[, value]`